### ISSUE TITLE
AVT3: JAWS read "none has been selected" when user actually selects Android or IOS in the select platform dropdown list

### ISSUE THREAD

**Issue by pankaj-kumar34** [carbon-design-system/carbon#6840]
=====ENVIRONMENT=====
Browser: chrome
IBM CI 162 Checkpoint Violation: IBM 1.3.1 Info and Relationships
IBM Guidance: https://www.ibm.com/able/guidelines/ci162/info_and_relationships.html

=====DEFECT DESCRIPTION=====
JAWS read "none has been selected" when user actually selects Android or IOS in the select paltforms dropdown list

1. Press tab to the 'Select platform' menu and press space to open the listbox
2. Press arrow key to the Android/IOS and press space to select Android/IOS

![image](https://user-images.githubusercontent.com/18532515/93211019-455ec480-f77e-11ea-966f-5994afbc73a4.png)

**Comment by dakahn** [comment]
Confirmed with both JAWS 2020 and NVDA

**Comment by deepika-bachu** [comment]
@dakahn - Can we have an update on this- this affects our product's accessibility compliance due to the dependency. Please look into this.

**Comment by dakahn** [comment]
No update to give. This is a high priority issue and a still a problem. @carbon-design-system/developers this fix will potentially involve a refactor of the component to use Downshift's `useMultipleSelection` hook. It's on our radar though, thanks for the ping.

**Comment by dakahn** [comment]
Circling back on this. On selection JAWS reads "undefined has been selected". If the list box is closed and then opened list items are not properly described as "selected".
